### ISSUE TITLE
Ctrl + S behaves like "Save and Exit" in embed mode

### ISSUE THREAD

**Issue by gowthamrajj** [jgraph/drawio#4010]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

If you are technical, you should reporting bugs along the lines of https://marker.io/blog/how-to-write-bug-report. If you are not technical, we will make allowances, please try to make an effort to understand the process.

**Describe the bug**
As per the description of JSON protocol in embed mode (https://www.drawio.com/doc/faq/embed-mode), exit value would be present for "Save and Exit" in post message. Save behaves correctly and doesn't have exit in the post message. But Ctrl + S behaves like "Save and Exit" includes exit: true in the post message. 

**To Reproduce**
1. Open draw.io in embed mode.
2. Press Ctrl + S.
3. The post message has exit: true

**Expected behavior**
Ctrl + S shouldn't send exit as true in post message or provide a way to differentiate between save operations and exit operation.

**Screenshots**
![Message event for save with exit:true](https://github.com/jgraph/drawio/assets/4691146/aac86200-346e-418f-b8c1-02b68a2276ea)

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 22.0.3

**Desktop (please complete the following information):**
 - OS: Mac OS
 - Browser: chrome
 - Browser Version: 119
 
 **Smartphone (please complete the following information):**
 - Device: NA
 - OS: NA
 - Browser NA
 - Version NA

**I tested the problem in incognito/private mode with all browser extensions switched off, write "yes" below:**
- yes

**Additional context**
I am embedding the draw.io app in our application to let users draw, but when users use the keyboard shortcut, due to exit:true being present in the post message, I am closing the iframe. But in confluence, Ctrl+S doesn't close draw.io, would like to have same behaviour. It might inconvenience the user if I close it on Ctrl + S.

**Comment by gowthamrajj** [comment]
Any help here please?

**Comment by alderg** [comment]
Are you showing the Save and Exit button in addition to the Save button?

**Comment by gowthamrajj** [comment]
> Are you showing the Save and Exit button in addition to the Save button?

Yes. But i have removed it for now and ignoring the exit: true in the postmessage.

**Comment by alderg** [comment]
If save and exit is shown, we're going to change the keyboard shortcut (and tooltips) for Ctrl+S and assign Ctrl+Shift+S to Save (which will deliver the save message without the exit flag). If save and exit is not shown, the exit flag will not be used. This keeps the protocol backwards compatible with the current implementations.
